.env.example

```diff
@@ -0,0 +1,11 @@
+#DB settings
+DB_VER=3.4
```

Comment:
Since this is pointed at indev, MongoDB should be `3.6`

Reply:
No issue here. I was using 3.6 at first but decided to downgrade to 3.4 *just-in-case* :)